src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/__tests__/HomeHighRiskCellConfiguratorTests.swift

```diff
@@ -71,17 +72,19 @@ class HomeHighRiskCellConfiguratorTests: XCTestCase {
 
 	func test_unknownRiskLevelCell_shouldBeEqual() {
 		let date = Date()
-		let configurator1 = HomeHighRiskCellConfigurator(isLoading: false, numberRiskContacts: 0, daysSinceLastExposure: 0, lastUpdateDate: date, manualExposureDetectionState: .waiting, detectionMode: .default, validityDuration: 0)
-		let configurator2 = HomeHighRiskCellConfigurator(isLoading: false, numberRiskContacts: 0, daysSinceLastExposure: 0, lastUpdateDate: date, manualExposureDetectionState: .waiting, detectionMode: .default, validityDuration: 0)
+
+		let configurator1 = HomeHighRiskCellConfigurator(isLoading: false, numberRiskContacts: 0, daysSinceLastExposure: 0, lastUpdateDate: date, manualExposureDetectionState: .waiting, detectionMode: .default, detectionInterval: 0)
+		let configurator2 = HomeHighRiskCellConfigurator(isLoading: false, numberRiskContacts: 0, daysSinceLastExposure: 0, lastUpdateDate: date, manualExposureDetectionState: .waiting, detectionMode: .default, detectionInterval: 0)
```

Comment:
it's really long parameters list. We might need some object holding them.

Reply:
configurator is already hold them and it is its duty to keep these properties. Introducing one more class just for holding properties could increase code complexity here.

Reply:
Hi @aleksandr-tikhonov ,
Thanks. I see. The xxxConfigurator is kinda mixture of ViewModel and some config logic. 
Yes, it's not necessary to bring in another class.